src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const task = [
+  {
+    description: 'Task1',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Task2',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Task3',
+    completed: false,
+    index: 3,
+  },
+
+];
+
+const list = document.querySelector('.list');
+
+function displayData() {
+  task.forEach((task) => {
+    const listElement = document.createElement('li');
+    listElement.classList.add('listElement');
+    listElement.innerHTML = '<div class = "checkbox"><input type = "checkbox" class= "taskCompleted"></div>';
+    listElement.innerHTML += `<div class = "listText">${task.description}</div>`;
+    listElement.innerHTML += '<div class = "delete"><img src="https://img.icons8.com/material-rounded/24/null/menu-2.png"/></div>';
+    list.appendChild(listElement);
+  });
```

Comment:
- [ ] You are very close to competing for this milestone 🚶 . However, there are some changes we need to apply to ensure that the lists are rendered as intended ☺️ . I have noticed the description is repeated and this is because it is being added to `listElement` twice. Kindly ensure to fix this so that the description rendered does not repeat 👍 .

Your outcome 😎 
![image](https://user-images.githubusercontent.com/99394374/220589098-57c9b188-8494-4e38-988d-1e2f4039cfe2.png)